docs/content/_snippets/operator/genesis.mdx

```diff
@@ -0,0 +1,19 @@
+<Tabs groupId="networks">
+
+<TabItem label="Testnet" value="testnet-genesis">
+
+```bash
+curl -fLJO https://dbfiles.testnet.iota.cafe/genesis.blob
+```
+
+</TabItem>
+
+<TabItem label="Devnet" value="devnet-genesis">
+
+```bash
+curl -fLJO https://dbfiles.devnet.iota.cafe/genesis.blob && curl -fLJO https://dbfiles.devnet.iota.cafe/migration.blob
+```
+
+</TabItem>
+
+</Tabs>
```

Comment:
We could only use devnet and testnet as value. Docusaurus will save this. So if people are looking for testnet stuff the tabs will be set to testnet everywhere. In adition we should use queryStrings so we redirect users to the correct tab with an query in the url. Wdyt?

````suggestion
<Tabs groupId="networks" queryString>

<TabItem label="Testnet" value="testnet">

```bash
curl -fLJO https://dbfiles.testnet.iota.cafe/genesis.blob
```

</TabItem>

<TabItem label="Devnet" value="devnet">

```bash
curl -fLJO https://dbfiles.devnet.iota.cafe/genesis.blob && curl -fLJO https://dbfiles.devnet.iota.cafe/migration.blob
```

</TabItem>

</Tabs>
````